source/gui/settingsDialogs.py

```diff
@@ -2531,6 +2531,26 @@ def onSave(self):
 		config.conf["documentFormatting"]["reportFrames"]=self.framesCheckBox.Value
 		config.conf["documentFormatting"]["reportClickable"]=self.clickableCheckBox.Value
 
+
+class ParagraphNavigationPanel(SettingsPanel):

```

Comment:
Should this really be a separate settings panel?

Could this go into "Review cursor" or "Document formatting"?

Reply:
I couldn't find a place for it that seemed reasonable. Believe me, I really didn't want another panel either. Where do you think it should go?

Reply:
After further discussions, a new panel is probably appropriate. Can it be named "Document Navigation" instead to be generic.

`detectFormatAfterCursor` might be better placed there as well.